pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -301,6 +309,13 @@ func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextM
 	}
 	genericConfig.ExternalAddress = url.Host
 
+	longRunningRequestRE := regexp.MustCompile(longRunningEndpointsRE)
+	longRunningRequestCheck := kgenericfilters.BasicLongRunningRequestCheck(longRunningRequestRE, nil)
+	kubeLongRunningFunc := genericConfig.LongRunningFunc
+	genericConfig.LongRunningFunc = func(r *http.Request) bool {
+		return longRunningRequestCheck(r) || kubeLongRunningFunc(r)
+	}
+
```

Comment:
@sttts i don't know if this is closer to what you had in mind... but i want to retain the behavior of delegating to the kubernetes default function so that if it is changed, we are still honoring it.  ptal.